bloom/generators/debian/templates/rules.em

```diff
@@ -59,4 +60,7 @@ override_dh_auto_install:
 	# in the install tree that was dropped by catkin, and source it.  It will
 	# set things like CMAKE_PREFIX_PATH, PKG_CONFIG_PATH, and PYTHONPATH.
 	if [ -f "@(InstallationPrefix)/setup.sh" ]; then . "@(InstallationPrefix)/setup.sh"; fi && \
-	dh_auto_install
+	dh_auto_install --destdir=$(CURDIR)/debian/@(Package)
```

Comment:
No, CURDIR isn't strictly necessary. The dbg package will happily build even if a relative path is used. I used CURDIR for consistency with the rest of the rules (ie: the override_dh_shlibdeps target uses `$(CURDIR)/debian`). We can switch to a relative path if that's what is recommended. I've seen CURDIR used fairly often in rules files though.